lib/proxies/iot.rest.js

```diff
@@ -188,6 +188,21 @@ IoTKitRestCloud.prototype.getCatalog = function (data, callback) {
 
     });
 };
+
+IoTKitRestCloud.prototype.pullActuations = function (data, callback) {
+    var me = this;
+    me.logger.info("Starting Actuations pulling ");
+    me.client.admin.pullActuations(data, function (err, response) {
+        if (!err && response) {
+            me.logger.info("Response From Actuations pull Retrieved ", JSON.stringify(response));
```

Comment:
change this logger line level to debug
add line:
me.logger.info("Actuations pulled successfully");